configs/providers.js

```diff
@@ -1,7 +1,7 @@
 module.exports = [
 	{
 		name: "RIBBON",
-		domain: "https://bins.ribbon.co",
+		domain: "http://bins.ribbon.co",
```

Comment:
Ribbon appears to be Payout now with bins.ribbon.co redirecting to bins.payout.com.  The ssl certificate for bins.ribbon.co is no longer valid - there also appears to be a 1000 request daily limit now as well.